lib/lambda.js

```diff
@@ -96,6 +138,15 @@ function elasticApmAwsLambda (agent) {
 
     return function wrappedLambda (payload, context, callback) {
       log.trace({ awsRequestId: context && context.awsRequestId }, 'lambda: fn start')
+
+      if (isColdStart) {
```

Comment:
Just noting that https://github.com/elastic/apm-agent-nodejs/pull/2194 is also detecting a coldstart -- whomever gets in second will want to make sure their code still plays nice with the other PR.

Reply:
Yup, noted. Let me know if you have a preference. I have the smaller diff here, so I'm happy to wait and handle the merge conflict, if you prefer.